Tests/test_file_dds.py

```diff
@@ -152,7 +152,7 @@ def test_sanity_ati2_bc5u(image_path: str) -> None:
 
 
 @pytest.mark.parametrize(
-    ("image_path", "expected_path"),
+    "image_path,expected_path",
```

Comment:
The suggestion in https://github.com/akx/Pillow/pull/7 is with spaces after the comma:

> * Replaced `("lut_mode", "table_size")` with `"lut_mode, table_size"`. It seems simpler to me, and is [the style that Pillow is using elsewhere](https://github.com/python-pillow/Pillow/blob/11b4df3ff9c02757eaf687ac13374f3525c8cb71/Tests/test_file_tiff_metadata.py#L195)

Can the formatter do it like that too?

```suggestion
    "image_path, expected_path",
```

Reply:
The autofix would not (did not) add a space, but the linter check is fine with strings that have or don't have spaces.

Re-pushed with spaces :)